src/app/index.js

```diff
@@ -86,8 +66,8 @@ export default class CGUs extends events.EventEmitter {
       throw error;
     };
 
-    this.trackDocumentChangesQueue.error(queueErrorHandler(true));
-    this.refilterDocumentsQueue.error(queueErrorHandler(false));
+    this.trackDocumentChangesQueue.error(queueErrorHandler);
```

Comment:
It will not create an issue on GitHub during the refiltering step because the Github module is attached to the process only after refiltering is done.